pkg/ccl/changefeedccl/changefeedbase/options.go

```diff
@@ -76,6 +76,7 @@ const (
 	OptAvroSchemaPrefix         = `avro_schema_prefix`
 	OptConfluentSchemaRegistry  = `confluent_schema_registry`
 	OptCursor                   = `cursor`
+	OptCustomKeyColumn          = `key_column`
```

Comment:
should this option support composite keys? (coma separate list of columns?)

Reply:
It's not needed for the motivating use case right now, and I think longer-term we'll go back to doing this via expressions.

Reply:
I'm readily waiting for a feature request to support this. :)